cmd/waku/server/rest/relay.go

```diff
@@ -215,3 +227,94 @@ func (r *RelayService) postV1Message(w http.ResponseWriter, req *http.Request) {
 
 	writeErrOrResponse(w, err, true)
 }
+
+func (r *RelayService) deleteV1AutoSubscriptions(w http.ResponseWriter, req *http.Request) {
+	var cTopics []string
+	decoder := json.NewDecoder(req.Body)
+	if err := decoder.Decode(&cTopics); err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+	defer req.Body.Close()
+
+	err := r.node.Relay().Unsubscribe(req.Context(), protocol.NewContentFilter("", cTopics...))
+	if err != nil {
+		r.log.Error("unsubscribing from topics", zap.Strings("contentTopics", cTopics), zap.Error(err))
+	}
+
+	writeErrOrResponse(w, err, true)
+}
+
+func (r *RelayService) postV1AutoSubscriptions(w http.ResponseWriter, req *http.Request) {
+	var cTopics []string
+	decoder := json.NewDecoder(req.Body)
+	if err := decoder.Decode(&cTopics); err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+	defer req.Body.Close()
+
+	var err error
+	_, err = r.node.Relay().Subscribe(r.node.Relay().Context(), protocol.NewContentFilter("", cTopics...))
+	if err != nil {
+		r.log.Error("subscribing to topics", zap.Strings("contentTopics", cTopics), zap.Error(err))
+	}
+
+	writeErrOrResponse(w, err, true)
+}
+
+func (r *RelayService) getV1AutoMessages(w http.ResponseWriter, req *http.Request) {
+	cTopic := chi.URLParam(req, "contentTopic")
+	if cTopic == "" {
+		w.WriteHeader(http.StatusBadRequest)
+		return
```

Comment:
Should we write that the contentTopic is missing? 
`_, err = w.Write([]byte("contentTopic is required"))`